lib/tasks/sync_checks.rake

```diff
@@ -27,8 +27,8 @@ namespace :sync_checks do
       content_item["schema_name"] == format
     end
 
-    checker.add_expectation("document_type") do |content_item, _|
-      content_item["document_type"] == format
+    checker.add_expectation("document_type") do |content_item, edition|
+      content_item["document_type"] == edition.artefact.kind
```

Comment:
So in light of artefacts not updating 'kind' when the format changes, won't this fail for some Editions?